gulpfile.js

```diff
@@ -42,6 +43,12 @@ gulp.task('fonts', function() {
     .pipe(gulp.dest('./dist/fonts'));
 });
 
+gulp.task('webpack-token-worker', function() {
```

Comment:
We need to add this as dependency to 'serve' task as otherwise we need to run it separately